.github/actions/update-axe-core-v1/src/run.ts

```diff
@@ -11,14 +11,16 @@ export default async function run(
 ) {
   try {
     const {
-      stdout: latestAxeCoreVersion,
+      stdout: npmInfoOut,
       stderr: npmInfoError,
       exitCode: npmExitCode
     } = await getExecOutput('npm', ['info', 'axe-core', 'version'])
     if (npmExitCode) {
       throw new Error(`Error getting latest axe-core version:\n${npmInfoError}`)
     }
 
+    // `npm info`` adds newline character to end of the output
```

Comment:
While testing saw that the build still created a pr even when versions were up-to-date. Turns out that the `stdout` of `npm info` had a newline character at the end so the check for equality failed and still would try to install axe-core@latest